### DISCUSSION TITLE
Woooooow 🤩

### DISCUSSION THREAD

**Discussion by lindamacdonald** [motiondivision/motionone#82] [Support]
This is what i have been waiting for for aaaaages 🤩 are there some tutorials for react?

**Comment by mattgperry** [comment]
I was actually about to write an article on this! My advice will be to prefer Framer Motion though haha. But if you wanna use Motion One with it it's broadly possible:

```jsx
function App() {
  const ref = useRef(null)
  useEffect(() => {
    animate(ref.current, { opacity: 1 })
  }, [])

  return <div ref={ref} style={{ opacity: 0 }} />
}
```

**Reply by lindamacdonald** [reply]
thanks a mil Matt :) ok then scratch my question on how to use motion one with react, just need to know how to use my motion one animations in framer motion then hehe :) sometimes I just get these wild animation ideas but dont quite know how to transform them into framer motion reality so a motion one/ framer motion tut would be awesome :) :) :)

**Comment by lindamacdonald** [comment]
and would love to see a tut on how to use the chrome extension too :) now you shall not here a peep from me for the rest of the evening because I shall be busy conjuring up all sorts of animations :)

**Comment by lindamacdonald** [comment]
scratch that last request, found the video :)
